unit2-java/intellij.rst

```diff
@@ -0,0 +1,30 @@
+Unit 2 (Java) - IntelliJ Learning Objectives
+=============================================
+
+Goals
+-----
+
+- Understand some of the tools and integrations that IntelliJ provides
+- Use IntelliJ to improve work efficiency and quality 
+
+Objectives
+----------
+
+Debugging
+^^^^^^^^^
+
+- Create breakpoints in source code
+- Run programs in debug mode
+- Inspect variables and parameters while in debug mode
+- Create conditional breakpoints
+
+IDE Features
+^^^^^^^^^^^^
+
+- Understand how IntelliJ uses Gradle to carry out like compiling and running code
+- Understand how IntelliJ flags compiler warnings and errors
+- User refactoring tools to efficiently improve and change code
+- Use embedded tools: terminal, Git, database interface 
```

Comment:
How about, "Use the embedded tools to carry out terminal commands, Git commands, and database interface."

Alternatively,
- Use embedded tools to carry out:
  - Terminal actions,
  - Git commands,
  - Database interface